editor/code_editor.cpp

```diff
@@ -321,40 +328,52 @@ void FindReplaceBar::_get_search_from(int &r_line, int &r_col) {
 }
 
 void FindReplaceBar::_update_results_count() {
-	if (results_count != -1) {
+	if (!needs_to_count_results && (result_line != -1)) {
+		results_count_to_current += (flags & TextEdit::SEARCH_BACKWARDS) ? -1 : 1;
+
+		if (results_count_to_current > results_count)
+			results_count_to_current = results_count_to_current - results_count;
+		else if (results_count_to_current == 0)
+			results_count_to_current = results_count;
```

Comment:
```suggestion
		if (results_count_to_current > results_count) {
			results_count_to_current = results_count_to_current - results_count;
		} else if (results_count_to_current == 0) {
			results_count_to_current = results_count;
		}
```

Reply:
Same below, make sure to always use brackets even for single-line `if`/`else`/`for`.

Reply:
Oh.... okiii 😊